src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -525,6 +527,33 @@ validate_args(#mrst{} = State, Args0) ->
     end.
 
 
+apply_limit(ViewPartitioned, Args) ->
+    LimitType = case ViewPartitioned of
+        true -> "partition_query_limit";
+        false -> "query_limit"
+    end,
+    
+    MaxLimit = config:get_integer("query_server_config",
+        LimitType, ?MAX_VIEW_LIMIT),
+
+    % set the highest limit possible if a user has not 
+    % specified a limit
+    Args1 = case Args#mrargs.limit =:= ?MAX_VIEW_LIMIT
```

Comment:
Super minor nit but I'd probably use `==` instead of `=:=` here. Only because I had to stop and think on whether we needed the stronger `=:=` for any reason or not.